packages/api/src/hooks/useServerTime.ts

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import useQuery from '../useQuery';
+import moment from 'moment';
+
+const epochToMoment = (epoch: number) => moment.unix(epoch).utc();
```

Comment:
nit: can we add some tsdocs to these functions

Reply:
actually just for the useServerTime

Reply:
I don't think this needs tsdocs thou, the function naming and signature is clear already 😅

Reply:
plus this is not used anywhere else, only inside useServerTime, buut I'll add some JSdocs for the `useServerTime` hook

Reply:
yupppp yeah i realised it as i wrote the comment 😆 just for useServerTime is fine

Reply:
I think it would be better to have these kind of helper functions inside utils folder

Reply:
true, since in P2P we will be using these quite a lot in other places like in orders page or checking how long the user is blocked for